ntropy_sdk/ntropy_sdk.py

```diff
@@ -1103,6 +1103,29 @@ def _add_transactions_list(
             model_name,
         )
 
+    @add_transactions.register(Iterable)
+    def _add_transactions_iterable(
+        self,
+        transactions,
+        timeout: int = 4 * 60 * 60,
+        poll_interval=10,
+        with_progress=DEFAULT_WITH_PROGRESS,
+        labeling=True,
+        create_account_holders=True,
+        model_name=None,
+        model=None,
+    ):
+        return self._add_transactions_list(
+            list(transactions),
```

Comment:
Would it possible to make this support iterables of huge length, so we would just get up to max_sync size chunks in a loop?

Reply:
In that case, should we return an iterator?